src/Common/src/TypeSystem/Common/TypeDesc.cs

```diff
@@ -456,5 +456,31 @@ public virtual MethodDesc GetFinalizer()
         {
             return null;
         }
+
+        public bool RequiresAlign8
```

Comment:
It is conceptually similar to ComputeRhCorElementType, so it should be wherever ComputeRhCorElementType lives.